components/mpas-ocean/src/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -3039,23 +3048,26 @@ subroutine ocn_init_validate_global_ocean(configPool, packagePool, iocontext, iE
       end if
 
       if (config_global_ocean_depress_by_land_ice) then
-         if (trim(config_global_ocean_land_ice_topo_file) == 'none') then
+         if (config_global_ocean_land_ice_topo_file == 'none' .and. &
+             config_global_ocean_topography_source == 'latlon_file') then
             call mpas_log_write( 'Validation failed for global ocean. '// &
                'Invalid filename for config_global_ocean_land_ice_topo_file', MPAS_LOG_CRIT)
             iErr = 1
             return
```

Comment:
Previously, you had to provide a placeholder name for `config_global_ocean_land_ice_topo_file` even if you used `config_global_ocean_topography_source = 'mpas_variable'`.  With this change, you should no longer get an error when you don't provide a placeholder filename.